solvers/BMI/util/inPolyhedron.m

```diff
@@ -0,0 +1,13 @@
+function flag = inPolyhedron(verts,pt)
```

Comment:
We do not have one function in drake/util doing this. But with functions in `thirdParty/polytopes` that I am going to merge in, it will be trivial to do this check